atom/browser/api/atom_api_window.cc

```diff
@@ -858,6 +864,7 @@ void Window::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("unhookWindowMessage", &Window::UnhookWindowMessage)
       .SetMethod("unhookAllWindowMessages", &Window::UnhookAllWindowMessages)
       .SetMethod("setThumbnailClip", &Window::SetThumbnailClip)
+      .SetMethod("setThumbnailTooltip", &Window::SetThumbnailTooltip)
```

Comment:
The method on `Tray` is called [`setToolTip`](http://electron.atom.io/docs/all/#traysettooltiptooltip) so I wonder if this should be `setThumbnailToolTip` for consistency.